datahub-web-react/src/app/entity/dataJob/DataJobEntity.tsx

```diff
@@ -22,6 +22,24 @@ const getDataJobPlatformName = (data?: DataJob): string => {
     return data?.dataFlow?.platform?.properties?.displayName || data?.dataFlow?.platform?.name || '';
 };
 
+const getExpandedNameForDataJob = (entity: DataJob): string => {
+    const name = entity?.properties?.name;
```

Comment:
what do you think about doing `const name = new DataJobEntity().displayName(entity);` in order to keep getting display names for dataJobs consistent?